### PR TITLE
Added logging to AWSParameterStorePropertySource

### PR DESCRIPTION
**Pull request by cschroed-usgs**
Addresses #509 and #527 

public domain

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@cschroed-usgs Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-aws&pullRequestId=529)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-aws&pullRequestId=529) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by cschroed-usgs** [comment]
According to https://cla.pivotal.io/about#obvious-fixes , contributions of "Logging messages or debug output" don't require signing a CLA.

**Comment by cschroed-usgs** [comment]
@pivotal-issuemaster This is an Obvious Fix

**Comment by pivotal-issuemaster** [comment]
@cschroed-usgs Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-aws&pullRequestId=529)!

**Comment by cschroed-usgs** [comment]
I signed the CLA anyway because I'm a US federal employee and [the same page](https://cla.pivotal.io/about#government-employee) says I should.
> If you are a government employee, please sign the CLA and add “public domain” to the pull request description.

File: spring-cloud-aws-parameter-store-config/src/main/java/org/springframework/cloud/aws/paramstore/AwsParamStorePropertySource.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ public void init() {
 
 	@Override
 	public Object getProperty(String name) {
+		logger.trace("getting property: " + name);
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-parameter-store-config/src/main/java/org/springframework/cloud/aws/paramstore/AwsParamStorePropertySource.java:64]:
```suggestion
		logger.trace("Getting property: {}", name);
```

File: spring-cloud-aws-parameter-store-config/src/main/java/org/springframework/cloud/aws/paramstore/AwsParamStorePropertySource.java [hunk 2]
```diff
@@ -69,6 +70,7 @@ private void getParameters(GetParametersByPathRequest paramsRequest) {
 				.getParametersByPath(paramsRequest);
 		for (Parameter parameter : paramsResult.getParameters()) {
 			String key = parameter.getName().replace(context, "").replace('/', '.');
+			logger.debug("setting property: " + key);
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-parameter-store-config/src/main/java/org/springframework/cloud/aws/paramstore/AwsParamStorePropertySource.java:73]:
```suggestion
			logger.debug("Setting property: {}", key);
```

**Pull request closed without merging**